packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -135,9 +141,10 @@ export default {
                     SPECIFY_MODE: `Create a USS directory named "testDir" with mode "rwxrwxrwx" `
                 }
             }
-        },
+        }, 
         POSITIONALS: {
-            DATASETNAME: "The name of the data set that you want to create"
+            DATASETNAME: "The name of the data set that you want to create",
+            LIKEDATASETNAME: "Name of the dataset to be liked"
```

Comment:
If the likeDataSetName positional is removed, please remove this string as well.